src/screens/Teams.js

```diff
@@ -1,38 +1,126 @@
 import React, { Component, } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { StyleSheet, Text, View, Image } from 'react-native';
+import { StyleSheet, Button, TouchableOpacity, Text, View, Image, TextInput, Alert } from 'react-native';
 import theme from '../styles/theme.style.js';
 import ScreenContainer from '../containers/ScreenContainer';
 import { Title, Subtitle, TextBody } from '../containers/TextContainer.js';
 import MainContainer from '../containers/MainContainer.js';
 import JoinTeam from '../components/modals/JoinTeam.js';
 import ParticipantListItem from '../components/ParticipantListItem';
+import FilterIcon from '../assets/images/icons/filter.png';
+import ClassLabel from '../components/Label.js';
+import XIcon from '../assets/images/icons/x-icon.png';
+
 
 class Teams extends Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      className: '',
+      classEntered: false,
+      isReady: false,
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.showName = this.showName.bind(this);
+  }
+
+  handleClass = (text) => {
+    this.setState({ className: text});
   }
 
-  // Write functions here
+  handleSubmit = () => {
+      this.setState({isReady : !this.state.isReady});
+      this.className.clear();
+  }
+  showName = () => {
+    this.setState({classEntered : !this.state.classEntered})
+  } 
 
   render() {
+    let {isReady} = this.state;
+
+    const renderLabel = () => {
+      if(isReady) {
+        return <ClassLabel labelColor="#5089E9" labelIcon={XIcon}><LabelClassName>{this.state.className}</LabelClassName></ClassLabel>;
```

Comment:
can you add a function to handle the label when you press on the x to cancel the label? right now it turns grey and there's no way to remove the label.

<img src="https://user-images.githubusercontent.com/43888017/159092290-e8666db2-a2a0-44a5-9065-8f9692914bed.png" width="200" />